src/components/album/Album.js

```diff
@@ -53,23 +80,40 @@ function Album({ album, wishListId }) {
         <AddToWishlistButton album={album} wishListId={wishListId} />
       </ButtonGroup>
       {/* Dialog */}
-      <Dialog open={openDialog} onClose={handleCloseDialog}>
+      <Dialog
+        open={openDialog}
+        onClose={handleCloseDialog}
+        fullWidth={fullWidth}
+        maxWidth={maxWidth}
+      >
         <DialogActions>
           <Button onClick={handleCloseDialog} color="primary">
             X
           </Button>
         </DialogActions>
         <DialogContent>
-          <DialogContentText>
+          <DialogContentText  style={{
+          
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}>
             <div>
               <div>
                 Release Date: {new Date(album.releaseDate).toLocaleDateString()}
               </div>
               <div>Average Rating: {album.averageRating}</div>
             </div>
           </DialogContentText>
-          <AlbumPreview spotifyUrl={album.spotifyUrl} />
         </DialogContent>
+        <AlbumPreview
+          albumId={album._id}
+          spotifyUrl={album.spotifyUrl}
+          numOfReviews={album.numOfReviews}
```

Comment:
Since we're trying to avoid giving `AlbumPreview` information it doesn't need; let's not pass in this prop here. Instead, it probably makes more sense to give this information to `AlbumReviews` component below and have it render the empty state message if needed.